.github/workflows/deploy_tre_reusable.yml

```diff
@@ -218,7 +218,7 @@ jobs:
       - name: Deploy TRE Core
         uses: ./.github/actions/devcontainer_run_command
         with:
-          COMMAND: "TF_VAR_ci_git_ref=${{ inputs.ciGitRef }} TF_LOG=${{ secrets.TF_LOG }} make deploy-core"
+          COMMAND: "TF_VAR_ci_git_ref=${{ inputs.ciGitRef }} TF_LOG=${{ secrets.TF_LOG }} make deploy-core && make api-healthcheck"
```

Comment:
What about splitting the extra make command into a separate step?

Reply:
Yeah, I was on the fence about whether to split or not. Happy to split it out!

Reply:
Done